pixi.toml

```diff
@@ -0,0 +1,49 @@
+[project]
+name = "rerun"
+version = "0.9.0"
+description = "Add a short description here"
+authors = ["Jeremy Leibs <jeremy@rerun.io>"]
+channels = ["conda-forge"]
+platforms = ["linux-64", "osx-arm64", "win-64"]
+
+[tasks]
+
+codegen = "cargo codegen"
+
+py-build = "maturin develop --manifest-path rerun_py/Cargo.toml --extras=tests"
+py-test = { cmd = "python -m pytest -vv rerun_py/tests/unit", depends_on = [
```

Comment:
Is there any way to get auto-complete of the commands in pixi? Or even list them somehow? Right now this just feels like a worse version of `justfile`

Reply:
Maybe we should think of pixi as a way to _implement_ our dev env/build system/etc, but keep `justfile` as the day-to-day front end.

Reply:
`pixi task list` will list the tasks. Similar to `just` they have the clap completions generator, e.g.
```
eval "$(pixi completion --shell bash)"
```
However, it doesn't look like they've added support for task completions yet -- I just created an issue though.

> Right now this just feels like a worse version of justfile

It is better in some ways:
 - Tasks themselves are more portable by virtue of running through cross platform deno shell: https://prefix.dev/docs/pixi/advanced/advanced_tasks . This means tasks actually run on windows compared to our situation where `just` is rarely usable outside of mac/linux.
 - The fact that dependencies are managed for you instead of needing to maintain a fragile hand-crafted installation script.
 
 And it's worse in some others ways:
 - The `justfile` make-inspired syntax is cleaner for task definition than toml
 - Tab completion doesn't work for tasks.